lib/ecto/migration.ex

```diff
@@ -393,11 +393,15 @@ defmodule Ecto.Migration do
               comment: nil,
               options: nil
 
+    @type column :: atom | String.t() | {index_dir(), atom | String.t()}
+
+    @type index_dir :: :asc_nulls_first | :asc_nulls_last | :desc_nulls_first | :desc_nulls_last
```

Comment:
do we need to include `:asc` and `:desc`?

Reply:
Yes, I think so, for two reasons:

 1. This is what we do for [order_by](https://hexdocs.pm/ecto/Ecto.Query.html#order_by/3).
 2. Not all databases support the `_nulls_first` and `_nulls_last` variants, which means that for e.g. MySQL and MSSQL, `:asc` and `:desc` are the only valid options.

Reply:
Yes, let's do it.

Reply:
@Munksgaard  I think they are missing from this type definition. Or am I getting confused?

Reply:
@greg-rychlewski Oh, you're absolutely right. I misunderstood your original message, how embarrassing. I've updated the PR to add the missing `:asc` and `:desc` to the type.